pkg/webhooks/mutator/utils.go

```diff
@@ -0,0 +1,35 @@
+package mutator
+
+import (
+	"context"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kubevirt/hyperconverged-cluster-operator/api/v1beta1"
+	hcoutil "github.com/kubevirt/hyperconverged-cluster-operator/pkg/util"
+)
+
+func getHcoObject(cli client.Client, namespace string) (*v1beta1.HyperConverged, error) {
+	ctx := context.TODO()
```

Comment:
Please define the context as the first function parameter and use it.

Reply:
Will do.
FYI, the other webhooks are implemented in the same way (context isn't being used).

Reply:
@nunnatsa The ns mutator also calls this function, but it dismisses the context parameter.
You can see [here](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/pkg/webhooks/mutator/mutator.go#L97) how I'm simply using the same logic that already exists and following the current pattern.

Does this mean that the already code is wrong?
Should I also add the context for other webhooks as well?

Reply:
If you don't mind to fix it, please do. You're refactoring this code anyway.

Reply:
:+1: